test/integration/integration_test.go

```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/onsi/ginkgo"
 	"github.com/onsi/ginkgo/config"
 	. "github.com/onsi/gomega"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Nit: Not sure why the formatter does this.

Reply:
I don't know either.   We're probably using different versions of goformat, since there's nothing to stop that.